apps/public/public.go

```diff
@@ -62,6 +62,9 @@ type Options struct {
 	HTTPClient ops.HTTPClient
 
 	capabilities []string
+
+	// Default is Instance discovery is enabled, unless disabled by setting DisableInstanceDiscovery = true
+	DisableInstanceDiscovery bool
```

Comment:
Recommend not exporting this field, because applications have no reason to set it
```suggestion
	disableInstanceDiscovery bool
```